moveit_setup_assistant/src/tools/moveit_config_data.cpp

```diff
@@ -1052,8 +1052,26 @@ bool MoveItConfigData::output3DSensorPluginYAML(const std::string& file_path)
 bool MoveItConfigData::outputJointLimitsYAML(const std::string& file_path)
 {
   YAML::Emitter emitter;
+  emitter << YAML::Comment("joint_limits.yaml allows the dynamics properties specified in the URDF "
+                           "to be overwritten or augmented as needed");
+  emitter << YAML::Newline;
+
   emitter << YAML::BeginMap;
 
+  emitter << YAML::Comment("For beginners, we downscale velocity and acceleration limits.") << YAML::Newline;
+  emitter << YAML::Comment("Increase those values to 1.0 to exploit the full potential of the robot.");
```

Comment:
How about

> For beginners, we downscale velocity and acceleration by default.
> You can always specify higher scaling factors (<= 1.0) in your motion requests.
> Increase those values to 1.0 to always move at maximum speed.

?

The second sentence is rather misleading right now.

Reply:
Fixed